### ISSUE TITLE
Errors when trying to use middleware with webpack-dev-server

### ISSUE THREAD

**Issue by typerandom** [stormpath/express-stormpath#390]
There seems to be some compatibility issues when running the `express-stormpath` middleware with [`webpack-dev-server`](https://webpack.github.io/docs/webpack-dev-server.html).

We need to investigate why this happens and if we can easily fix it.

**Comment by robertjd** [comment]
Are you getting some errors or re-produceable behavior that is problematic?

**Comment by typerandom** [comment]
Yeah, basically it was complaining that `app.get` was undefined. But I haven't investigated this in-depth to have anything more to add. My thought with creating this issue was to track it so that we can investigate it more later on.

**Comment by bentedder** [comment]
What kind of errors are you getting? I've been using it with webpack-dev-server for a while now...
